src/applications/gi/components/vet-tec/VetTecFilterBy.jsx

```diff
@@ -54,6 +101,8 @@ class VetTecFilterBy extends React.Component {
           onChange={this.handleFilterChange}
           options={options}
         />
+        {/* prod flag for CT-116 - #19864 */}
+        {!environment.isProduction() && this.renderProviderFilters()}
```

Comment:
This will be removed in https://github.com/department-of-veterans-affairs/vets.gov-team/issues/19866